datasets/ba/companies/crawler.py

```diff
@@ -0,0 +1,522 @@
+            parsed_bd = list(map(str.strip, bd.split(" ,")))
+            # JD: Name might be Dioničari prema evidenciji Registra vrijednosnih papir F BiH
+            # Shareholders according to the records of the FBiH Securities Register
+            company_name = parsed_bd[0]
+
+            # includes country
+            address = ", ".join(parsed_bd[1:]).strip().strip(",")
+            country = ""
+            if len(parsed_bd) > 1:
+                country = parsed_bd[-1]
+
+            founders_companies.append(
+                {
+                    "basic_data": bd,
+                    "registration_number": (
+                        reg_num[i] if re.search(r"\d", reg_num[i]) else ""
+                    ),
+                    "name": company_name,
+                    "address": address,
+                    "country": country,
+                }
+            )
+
+        managers = []
+        managers_url = urljoin(
+            BASE_URL, founders_page.xpath('//*[@id="podmeni"]/p/a')[1].attrib["href"]
+        )
+        managers_page = context.fetch_html(managers_url, headers=HEADER)
+        managers_names = managers_page.xpath(
+            "//td[contains(text()," + ' "Name")]/' + "following-sibling::td/text()"
+        )
+
+        managers_pos = managers_page.xpath(
+            "//td[contains(text(),"
+            + ' "Position")'
+            + "]/following-sibling"
+            + "::td/text()"
+        )[1::2]
+
+        managers_auth = managers_page.xpath(
+            "//td[contains(text(),"
+            + ' "Authorisations/'
+            + ' Position limits")'
+            + "]/following-sibling::td/"
+            + "text()"
+        )
+        if managers_names:
+            for i, manager in enumerate(managers_names):
+                managers.append(
+                    {
+                        "name": manager,
+                        "authorizations": managers_auth[i],
+                        "position": managers_pos[i],
+                    }
+                )
+
+    finally:
+        entity = context.make("Company")
+        entity.id = context.make_id(
+            "BACompany", record["registration_number"], record["name"]
+        )
+
+        entity.add("name", record["name"], lang="bos")
+        entity.add("name", record["abbreviation"], lang="bos")
+        if record.get("status_bankruptcy"):
+            entity.add("status", record["status_bankruptcy"], lang="bos")
+
+        entity.add("country", "ba")
+        entity.add("address", record["address"], lang="bos")
+        if record["address_additional"]:
+            entity.add("address", record["address_additional"], lang="bos")
+
+        entity.add("legalForm", record["legal_form"], lang="bos")
+        entity.add("registrationNumber", record["registration_number"])
+
+        # JD: LegalEntity:idNumber probably?
+        if record.get("unique_id"):
+            entity.add("registrationNumber", record["unique_id"])
```

Comment:
damn, no support for Bosnia and Herzegovina yet in stdnum https://github.com/arthurdejong/python-stdnum/issues/268

yup, this one also looks like one `JIB	1234567890123	13 digits`

Reply:
Ok, what should I do? Ignore?

Reply:
Nope, this is perfect